### PR TITLE
feat: 🎸 HW-1 - Implemented first task. Yoohoo!

### PR DESCRIPTION
**Pull request by vladtihomirov**
## What was done:
1. Created a basic structure of your project
2. Edited **package.json** file: add necessary information about you and your project
3. Rendered an arbitrary list of items

## Screenshot
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+# SDC - Modern Web Development 2024: Homework #1
 
-### Setup
-- Replace all occurrences of `{{.HOMEWORK_NUMBER}}` with the number of the homework you are working on.
-- Paste task description in the `README.md` file.
-- Implement your solution.
\ No newline at end of file
+## Description
+1. Create new repository for react homeworks with an appropriate name
+2. Install and run **create react app**
```
Comment (anddrews) [inline README.md:5]: You use vite not CRA

File: src/services/DataService.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import {Item} from "../@types/Item.ts";
+
+export class DataService {
```
Comment (anddrews) [inline src/services/DataService.ts:3]: What is the reason for creating a  class with the static method?

Comment (vladtihomirov) [reply src/services/DataService.ts:3]: Isolating data (that should come from API) from component. Better be done as JSON and request but nothing like this was said in the task about it so I just put it here

File: README.md (hunk 1 above)
Comment (vladtihomirov) [reply README.md:5]: Copy and paste from you task before change

image

Comment (vladtihomirov) [reply README.md]: Resoleved

File: src/services/DataService.ts (hunk 2 above)
Comment (vladtihomirov) [reply src/services/DataService.ts]: resolved.
Made as normal service

**Pull request closed without merging**